google_photos_takeout_helper/__main__.py

```diff
@@ -462,7 +472,7 @@ def set_file_geo_data(file: Path, json):
 
     # Fixes ALL metadata, takes just file and dir and figures it out
     def fix_metadata(file: Path):
-        print(file)
+        #print(file)
```

Comment:
We probably neet to wrap the whole function in `try`, and print the file only in catch. I will do this when resolving #25

Reply:
Haha yeah I will leave this alone since it isn't related I just need the logs to be a little less spammy 😜.